vividus-extension-selenium/src/main/java/org/vividus/selenium/screenshot/ScreenshotTaker.java

```diff
@@ -16,9 +16,13 @@
 
 package org.vividus.selenium.screenshot;
 
+import java.awt.image.BufferedImage;
+import java.io.IOException;
 import java.util.Optional;
 
 public interface ScreenshotTaker
 {
     Optional<Screenshot> takeScreenshot(String screenshotName);
+
+    BufferedImage takeViewportScreenshot() throws IOException;
```

Comment:
usage of ScreenshotShootingStrategy is currently redundant for mobile app, also it's in common interface because it perfectly suites for both mobile and web platform

Reply:
If you use the strategy there is no need to support screenshot taking code

Reply:
I don't need strategy in mobile application, I just need to take a screenshot of viewport without any additional handling